src/Renderer/Label2DRenderer.js

```diff
@@ -136,16 +128,30 @@ class Label2DRenderer {
     }
 
     registerLayer(layer) {
-        this.domElement.appendChild(layer.domElement);
+        this.domElement.appendChild(layer.domElement.dom);
     }
 
     render(scene, camera) {
-        if (!this.infoTileLayer || !this.infoTileLayer.layer.attachedLayers.find(l => l.isLabelLayer && l.visible)) { return; }
+        const labelLayers = this.infoTileLayer && this.infoTileLayer.layer.attachedLayers.filter(l => l.isLabelLayer && l.visible);
+        if (labelLayers.length == 0) { return; }
         this.grid.reset();
 
         // set camera frustum
         frustum.setFromProjectionMatrix(camera.projectionMatrix);
-        this.culling(scene, this.infoTileLayer.displayed.extent, camera);
+
+        labelLayers.forEach((labelLayer) => {
```

Comment:
This maked the `Label2DRenderer` aware of the internal structure of `LabelLayer` and we will have to modify here if we change it. I think it would be better to implement a method in `LabelLayer` to get all the `LabelsNode` instead. WDYT?

Reply:
I add:
* `LabelLayer.submittedLabelNodes` to get all submitted nodes for rendering;
* `LabelsNode.labels` to get all labels's node;

Reply:
What is `submittedLabelNodes` ? Is it the nodes that `needsUpdate` ? Or visible labels ? Something else?

Reply:
it's all label nodes submitted to the render processing